libs/sdk-backend-bear/src/convertors/dateFormatting/dateFormatter.ts

```diff
@@ -10,37 +10,37 @@ import { createDateValueFormatter } from "./dateValueFormatter";
 import { DateFormat, DEFAULT_DATE_FORMAT } from "./dateValueParser";
 import { createDefaultDateFormatter } from "./defaultDateFormatter";
 
-export function findDateAttributeUri(dimensions: IDimensionDescriptor[]): string | undefined {
-    const dateAttrDescriptor = dimensions
+export function findDateAttributeUris(dimensions: IDimensionDescriptor[]): string[] {
```

Comment:
please mark this fun as TODO: rewrite. move search for date attributes into data view facade / meta() section

Reply:
done